spec/ruby/language/kernel_spec.rb

```diff
@@ -0,0 +1,11 @@
+require_relative '../spec_helper'
+
+describe "The Kernel class" do
+  it "inspects an object without `#class` method" do
+    obj = Object.new
+    class << obj
+      undef_method :class
+    end
+    obj.inspect.should be_kind_of(String)
+  end
```

Comment:
Could you move this spec to https://github.com/oracle/truffleruby/blob/master/spec/ruby/core/kernel/inspect_spec.rb ?